src/main/java/com/mineinabyss/mobzy/ecs/systems/ModelEngineSystem.kt

```diff
@@ -10,7 +10,7 @@ import org.bukkit.event.Listener
 
 
 object ModelEngineSystem : Listener {
-    private val modelManager = runCatching { ModelEngineAPI.api.modelManager }.getOrNull()
+    private val modelManager by lazy { runCatching { ModelEngineAPI.api.modelManager }.getOrNull() }
```

Comment:
What exactly is this fixing? I remember specifically not using lazy so we dont have to do `?: return` constantly

Reply:
modelManager was null if we just did the runCatching method